src/content/docs/logs/forward-logs/heroku-log-forwarding.mdx

```diff
@@ -16,36 +16,34 @@ redirects:
 
 import addData from 'images/add-data.png'
 
-import herokuTile from 'images/heroku-tile.png'
```

Comment:
Please remove this file as well, as (I believe) it is not used anymore.

Reply:
It's being used by translations, let me ask how to proceed.

Reply:
yup! You're right! 🔥